token/js/src/state/account.ts

```diff
@@ -127,6 +111,51 @@ export async function getAccount(
     };
 }
 
+/**
+ * Retrieve information about multiple token accounts in a single RPC call
+ *
+ * @param connection Connection to use
+ * @param addresses  Token accounts
+ * @param commitment Desired level of commitment for querying the state
+ * @param programId  SPL Token program account
+ *
+ * @return Token account information
+ */
+export async function getMultipleAccounts(
```

Comment:
nit: possible to move this after `getAccount`? Just looking to have the file flow logically